Simplenote/NoteListViewController+Swift.swift

```diff
@@ -308,18 +308,11 @@ extension NoteListViewController: EditorControllerNoteActionsDelegate {
 //
 extension NoteListViewController: EditorControllerSearchDelegate {
 
-    public func editorControllerDidBeginSearch(_ controller: NoteEditorViewController) {
-        SPTracker.trackListNotesSearched()
-    }
-
     public func editorController(_ controller: NoteEditorViewController, didSearchKeyword keyword: String) {
+        SPTracker.trackListNotesSearched()
         searchKeyword = keyword
         refreshPredicate()
     }
```

Comment:
Just an observation: tracking will be different as it looks like we will now send an event on every keypress

Reply:
Aye, for the record, changed on purpose (since this API is invoked with a delay, that is, everytime a word is compelte).

PLUST I wanted to simplify the whole thing. Thank you for pointing out sir!!